### PR TITLE
List most significant (>1k stars) meta repos

### PR DESCRIPTION
**Pull request by nicolas-grekas**
This is not an advertising list :)
Many projects do have a main "meta" repository, from which you can use only a subpart (components).
Here is the list I computed from those with more than 1k stars. Sorry in advance if I forgot anyone, PR welcomed of course.
(e.g. "thephpleague" is not listed as they don't have one main repo AFAIK)

### CODE REVIEW

File: src/Command/ThanksCommand.php [hunk 1]
```diff
@@ -24,13 +24,23 @@
...
     private static $mainRepositories = [
         'api-platform' => [
             'name' => 'api-platform/api-platform',
             'url' => 'https://github.com/api-platform/api-platform',
         ],
+        'cakephp' => [
+            'name' => 'cakephp/cakephp',
+            'url' => 'https://github.com/cakephp/cakephp',
+        ],
+        'doctrine' => [
```
Comment (stof) [inline src/Command/ThanksCommand.php]: I'm not sure about this one. Doctrine packages are not subparts of the `doctrine/doctrine2`. they are separate Doctrine projects (and so, if you use anything using the ORM, it is already thanked)

File: src/Command/ThanksCommand.php [hunk 2]
```diff
@@ -39,10 +49,42 @@ class ThanksCommand extends BaseCommand
...
+        ],
+        'slimphp' => [
+            'name' => 'slimphp/Slim',
+            'url' => 'https://github.com/slimphp/Slim',
+        ],
+        'Sylius' => [
+            'name' => 'Sylius/Sylius',
+            'url' => 'https://github.com/Sylius/Sylius',
+        ],
         'symfony' => [
```
Comment (stof) [inline src/Command/ThanksCommand.php]: should we rather handle it in the `extra` key of Symfony subtree packages ? We have separate Symfony packages, which are separate from `symfony/symfony`

File: src/Command/ThanksCommand.php (hunk 1 above)
Comment (nicolas-grekas) [reply src/Command/ThanksCommand.php]: it has been removed since this PR, for the reason you listed :)

File: src/Command/ThanksCommand.php (hunk 2 above)
Comment (nicolas-grekas) [reply src/Command/ThanksCommand.php]: that's a "second scale" detail, `symfony/*` (whether from subtree splits or not) are not wrong forwarding to `symfony/symfony` (same reasoning for other entries in this list)

Comment (stof) [reply src/Command/ThanksCommand.php]: well, `symfony/polyfill-*` should rather forward to `symfony/polyfill` instead, as this is where this code comes from.

**Pull request merged**